src/components/attribution-control.ts

```diff
@@ -21,7 +21,7 @@ export type AttributionControlProps = {
 
 function AttributionControl(props: AttributionControlProps): null {
   const ctrl = useControl<MapboxAttributionControl>(
-    ({mapLib}) => new mapLib.AttributionControl(props),
+    ({mapLib}) => new mapLib.AttributionControl(props) as MapboxAttributionControl,
```

Comment:
Can we move to a lib-neutral type overtime on these as well? Seeing `Mapbox{type}` when using maplibre is confusing to the uninitiated.